PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,9 +11,12 @@ Add any additional information that might be relevant to reviewers.
 
 ## Checklist
 - [ ] I have read and followed the [contribution guidelines](README.md#contributing) for this project.
-- [ ] I have added tests (if applicable).
+- [ ] I have added positive and negative test scenarios (if applicable)
 - [ ] All tests are passing.
 - [ ] I have updated the documentation (if necessary).
+- [ ] The pull request title follows the standard: [Pull Request Naming Conventions](https://github.com/afet-yonetim-sistemi/ays-be-api-automation/blob/main/CONTRIBUTING.md#pull-request-naming-conventions).
+- [ ] The branch name follows one of the conventions outlined in the [Branch Naming Conventions](https://github.com/afet-yonetim-sistemi/ays-be-api-automation/blob/main/CONTRIBUTING.md#branch-naming-conventions).
+
 
 ## Reviewer Instructions
 Provide instructions or guidelines for reviewers to follow when assessing your pull request.
```

Comment:
@Nbhtozcn selamlar, Pull Request Template dosyasını BE repoda olduğu şekilde düzenleyelim diye konuşmuştuk diye hatırlıyorum ama bu şekilde kalması yönünde mi düşündünüz? BE taraftaki daha kapsamlı ve düzenli görünüyor diye bir sormak istedim.

Reply:
Selam Çağla, aslında be reposunda farklı olarak sadece check pointleri tek cümle halinde yazmak yerine madde madde yazıp altına 1-2 madde halinde açıklama yapilmis. Coverage acısından bi fark göremediğim icin bu sekilde bıraktım ama onun gorunumu daha iyi derseniz o sekilde update edebilirim. Ya da su kısma değinilmemiş o da eklensin dediğiniz bir nokta varsa onu da ekleyebilirim.

Reply:
Böylede kalabilir benim için sorun yok açıkçası ama proje genelinde bir bütünlük olsun isteniliyorsa benzer şekilde düzenlemek daha iyi olabilir belki. 
Ek olarak code kalitesi ve SonarLint'te herhangi bir sorun olmamasının kontrol edilmesini eklesek güzel olur. Çünkü bazen bir test yazarken bazı şeyleri değiştirdiğimizde import statement kısmında eklenen kısmı silmeyi unutmuş olabiliyoruz onu SonarLint hemen gösteriyor sarı bir bayrak uyarısı vererek. 
Birde testlere Smoke ve Regression taglarini ve xml dosyasına eklenmesini sağlayabilmek için Smoke ve Regression testleri yazılmış ve dahil edilmiştir şeklinde listeye ekleyebiliriz.
Elinize emeğinize sağlık

Reply:
Selamlar, bizim framework'teki template de ayni seylere değindiği icin benim icin de sorun yok acikcasi gayet net ve anlasilir duruyor bana gore ama dediğiniz gibi be reposundaki sekilde yapılması istendiyse eğer title ve maddeler seklinde, o zaman değiştirilebilir.

Reply:
Açıklayıcı ve kapsamlı görünüyor. BE template ile hemen hemen aynı gibi. Code kalitesi ve SonarLint kısmında Çağla'ya katılıyorum double check yapmak ve gözden kaçmayı en aza indirmek için gerekli bir adım bence.

Reply:
@cagla88 @htczrrsz @Emelpoyraz Son commit ile daha iyi oldu diye düşünüyorum. Atladigim bir yer varsa paylaşın lütfen, teşekkürler.

Reply:
@Nbhtozcn çok iyi olmuş emeğinize sağlık